po/zh-CN.po

```diff
@@ -1941,25 +1956,30 @@ msgid ""
 "feature\n"
 "and we should show students this right away."
 msgstr ""
+"第一天的主要目标是要谈到著名的borrow checker，其他方面点到为止。Rust处理内存的方式是其主要特点，这"
+"点我们应该尽早展示给学生。"
 
 #: src/welcome-day-1.md:33
 msgid ""
 "If you're teaching this in a classroom, this is a good place to go over the\n"
 "schedule. We suggest splitting the day into two parts (following the slides):"
 msgstr ""
+"如果你是在教室里教授此课程，不妨在这里介绍一下时间安排。 这边建议是把每天分成两部分（跟着PPT来）："
```

Comment:
```suggestion
"如果你是在教室里教授此课程，不妨在这里介绍一下时间安排。 这边建议是把每天分成两部分（跟着幻灯片来）："
```

Reply:
> I assume "PPT" is a more widely used term to refer to this form of content.

Wow! :smile: I don't know anything about Chinese, but I find it interesting if people call all slideshows "PPT" after the PowerPoint file extension.

Reply:
> However, strictly speaking I think "PPT" can only refer to the PowerPoint file type, which may not exactly fit our case.

This is absolutely correct, that's why I mentioned above "幻灯片" is the "correct" translation. 
You also agreed that, "in daily life slides are usually referred as "PPT" in Chinese conversations.", so I think it's at least an acceptable, if not better, translation.
 
Ultimately I think this is a question for @mgeisler, to decide on the content style (orthodox VS laid-back), and the how far we want to go on localization.

Since we already used 幻灯片 in instances, I'll stick to it for consistency. But maybe we should update all its occurrences to PPT. :P 


And thanks @superwhd for your thorough review!

Reply:
> Ultimately I think this is a question for @mgeisler, to decide on the content style (orthodox VS laid-back), and the how far we want to go on localization.

I'm not an expert in localization, but my personal style would be an informal and laid-back style. Since others might have thought about the same, I started #818 to discuss this. I don't think all languages need to use the same style: if someone tells me that educational material in language X always uses a formal style, then I'm completely happy to let the translators use that style :slightly_smiling_face: